collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1396,6 +1396,11 @@ public void onScreenChange(Direction direction) {
      */
     @Override
     public void onScreenRefresh() {
+        FormController formController = getFormController();
+        if (formController == null) {
+            Timber.w("FormController in the application class is null, FormController in formEntryViewModel is %s", (formEntryViewModel.isFormControllerSet() ? "not null" : "null"));
```

Comment:
Here I want to know if both formControllers are null or maybe there is only something wrong with the one stored in the application class.